doc/source/development/contributing.rst

```diff
@@ -24,6 +24,16 @@ and `good first issue
 where you could start out. Once you've found an interesting issue, you can
 return here to get your development environment setup.
 
+When you start working on an issue, it's a good idea to assign the issue to yourself,
+so nobody else duplicates the work on it. GitHub restricts assigning issues to maintainers
+of the project only. Until recently, contributors added a comment letting others know they
+are working on an issue. But we implemented a workaround consisting of adding
+a comment with the exact text `take`. When you do it, a GitHub action will automatically assign
+you the issue few seconds later. This way, it's possible to filter the list of issues, and find
+only the unassigned ones. Work on assigned issues can be discontinued. Feel free to also
+consider assigned issues. Just kindly ask the current assignee if you can take an issue,
+if it's been inactive for some time (a week at least).
```

Comment:
Does this action prevent people from assigning themselves already-assigned issues?

I would also break this off into a separate paragraph, as this is a relatively important detail from an operations perspective.

Reply:
The action just adds to the list of the assignees the user who add a comment with the exact text `take`. If someone is already assigned, the new user is simply added, and the previous assignees keep unchanged. Happy to consider other options, but IMHO the simplest this works, the better.

Reply:
Do you mean creating a new section for this new paragraph, or would you split the paragraph in two?

Reply:
> If someone is already assigned, the new user is simply added, and the previous assignees keep unchanged.

Got it.  That's fine for now.  Let's see how it plays out.

> Do you mean creating a new section for this new paragraph, or would you split the paragraph in two?

Split into two.

Reply:
Thanks. I splitted it and expended it a bit. Let me know what do you think. Thanks!

Reply:
I like the split.  Thanks!